_posts/05-03-01-Date-and-Time.md

```diff
@@ -6,59 +6,65 @@ anchor: date_and_time
 
 ## Datum i vreme {#date_and_time_title}
 
-PHP ima klasu sa nazivom DateTime koja služi da vam pomogne kada čitate, pišete, poredite ili računate datume ili vreme.
-Postoji mnogo funkcija koje se odnose na datum i vreme u PHP-u pored DateTime, ali ona pruža dobar objektno-orijentisani
-interfejs za najčešće upotrebe. Može da rukuje vremenskim zonama, ali to je van opsega ovog kratkog uvoda.
+PHP poseduje DateTime klasu za potrebe čitanja, pisanja, poređenja i računanja datume i vremena.
+Postoji i dosta drugih funkcija za rad sa datumom/vremenom u PHP-u pored DateTime klase, ali ona
+pruža dobar objektno-orijentisani interfejs za najčešće slučajeve korišćenja. Ova klasa može da
+manipuliše i vremenskim zonama, ali to je van okvira ovog kratkog uvoda.
+
+Da biste započeli sa radom sa DateTime klasom, konvertujte string koji sadrži datum i vreme u objekat pomoću
+`createFromFormat()` factory metode ili pozovite `new \DateTime` kako biste kreirali trenutni datum/vreme.
+Koristite `format()` metodu za konvertovanje DateTime instance u string.
 
-Da biste započeli sa radom sa DateTime, konvertujte sirov string koji sadrži datum i vreme u objekat pomoću
-`createFromFormat()` factory metode ili stavite `new \DateTime` da dobijete trenutni datum i vreme. Koristite `format()`
-metodu da konvertujete DateTime nazad u string za izlaz.
 {% highlight php %}
 <?php
 $raw = '22. 11. 1968';
-$start = \DateTime::createFromFormat('d. m. Y', $raw);
+$start = DateTime::createFromFormat('d. m. Y', $raw);
 
-echo 'Start date: ' . $start->format('m/d/Y') . "\n";
+echo 'Početni datum: ' . $start->format('Y-m-d') . "\n";
 {% endhighlight %}
 
-Računanje sa DateTime je moguće korišćenjem klase DateInterval. DateTime ima metode kao `add()` and `sub()` koje primaju
-DateInterval kao argument. Nemojte pisati kod koji pretpostavlja isti broj sekundi svakog dana, i letnje i zimsko
-pomeranje vremena i promene vremenske zone će oboriti tu pretpostavku. Umesto toga koristite vremenske intervale. Za
-računanje razlike u datumima koristite metodu `diff()`. Ona vraća new DateInterval, što je vrlo lako prikazati.
+Računanje je moguće korišćenjem DateInterval klase. DateTime ima metode kao što su `add()` and `sub()` koji prihvataju
+DateInterval instancu kao argument. Nemojte pisati kôd koji očekuje isti broj sekundi svakog dana, jer će letnje i zimsko
+računanje vremena i promene vremenske zone oboriti tu pretpostavku. Umesto toga koristite vremenske intervale. Za
+računanje razlike između datuma koristite metodu `diff()`. Ona vraća DateInterval instancu, koja u sebi sadrži sve
```

Comment:
Mislim da ovde ima odstupanja od originalnog teksta:
> It will return new DateInterval, which is super easy to display.

_Ona vraća DateInterval instancu, koja u sebi sadrži sve potrebne informacije._